src/stability_sdk/animation.py

```diff
@@ -133,20 +133,15 @@ class DepthSettings(param.Parameterized):
 
 class Rendering3dSettings(param.Parameterized):
     camera_type = param.ObjectSelector(default='perspective', objects=['perspective', 'orthographic'])
-    image_render_method = param.ObjectSelector(default='mesh', objects=['pointcloud', 'mesh'])
-    # Mask render method is selected based on a type of model (inpainting/non-inpainting) used for inpainting for current frame
-    image_render_points_per_pixel = param.Integer(default=8)
-    image_render_point_radius = param.Number(default=0.006)
-    image_max_mesh_edge = param.Number(default=0.1)
-    mask_render_points_per_pixel = param.Integer(default=4)
-    mask_render_point_radius = param.Number(default=0.0045)
-    mask_max_mesh_edge = param.Number(default=0.04)
-
+    image_render_mode = param.ObjectSelector(default='mesh', objects=['pointcloud', 'mesh'])
+    mask_render_mode = param.ObjectSelector(default='mesh', objects=['pointcloud', 'mesh'])
+    
 class InpaintingSettings(param.Parameterized):
     non_inpainting_model_for_diffusion_frames = param.Boolean(default=False, doc="If True, for each diffusion frame, inpainting will be conducted using regular non-inpainting model to optimize number of generations.")
     inpaint_border = param.Boolean(default=False, doc="Use inpainting on top of border regions for 2D and 3D warp modes. Defaults to False")
     do_mask_fixup = param.Boolean(default=True, doc="Enforce pixels outside of inpainting region to be equal to original frame")
     mask_min_value = param.String(default="0:(0.1)", doc="Mask postprocessing for non-inpainting model. Mask floor values will be clipped by this value prior to inpainting")
+    mask_binarization_thr = param.Number(default=0.1, softbounds=(0,1), doc="Applied when inpainting with inpainting model. Grayscale mask values lower than this value will be set to 0, values that are higher — to 1.")
```

Comment:
`mask_threshold` may be simpler name. I think it can be assumed when a threshold is applied it is for binarization

Reply:
I might sort of prefer the clumsy `mask_binarization_thr` name (in comparison to `mask_threshold`). The inpainting vs. non-inpainting model for inpainting.. and sharing the same settings.. is a little confusing and I expect that any kind of prefix to hint that it's only conditionally relevant would tend to be valuable to some users. Hot-take: Maybe selective clumsiness can be a great signal. (don't quote me on this - it just sounded cool in my head 😄)